jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -211,7 +211,7 @@ YUI.add('env-switcher', function() {
     */
     showUserProfile: function() {
       this.props.changeState({
-        sectionC: {
+        sectionB: {
           component: 'profile',
```

Comment:
I changed the section the profile is displayed in, but forgot to update these changeState calls to use the new section.